x/evm/module.go

```diff
@@ -18,71 +22,87 @@ import (
 	"github.com/cosmos/ethermint/x/evm/types"
 )
 
-var _ module.AppModuleBasic = AppModuleBasic{}
-var _ module.AppModule = AppModule{}
+var (
+	_ module.AppModule      = AppModule{}
+	_ module.AppModuleBasic = AppModuleBasic{}
+)
 
-// AppModuleBasic struct
+// AppModuleBasic defines the basic application module used by the evm module.
 type AppModuleBasic struct{}
 
-// Name for app module basic
+// Name returns the evm module's name.
 func (AppModuleBasic) Name() string {
 	return types.ModuleName
 }
 
-// RegisterCodec registers types for module
-func (AppModuleBasic) RegisterCodec(cdc *codec.Codec) {
-	types.RegisterCodec(cdc)
+// RegisterLegacyAminoCodec performs a no-op as the evm module doesn't support amino.
+func (AppModuleBasic) RegisterLegacyAminoCodec(_ *codec.LegacyAmino) {
```

Comment:
should we remove this func? since it's doing nothing and return nothing as well.

Reply:
it's required by the module interface